cmd/odf/rook/purge_osd.go

```diff
@@ -0,0 +1,38 @@
+package rook
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+var forceFlag = "force"
+
+var rookPurgeOsdCmd = &cobra.Command{
+	Use:     "purge-osd",
+	Short:   "Permanently remove an OSD from the cluster.",
+	Args:    cobra.ExactArgs(1),
+	Example: "odf purge-osd <id>",
+	PreRunE: validateOsdID,
+	Run: func(cmd *cobra.Command, args []string) {
+		// ctx := cmd.Context()
+		// clientsets := root.GetClientsets(cmd.Context())
+		// forceflagValue := cmd.Flag(forceFlag).Value.String()
+
+		//rook.PurgeOsd(ctx, clientsets, root.OperatorNamespace, root.StorageClusterNamespace, args[0], forceflagValue)
```

Comment:
The `--prompt-if-not-safe` flag needs to be passed here unless they specified the force flag.

Reply:
But perhaps we don't even need to support the force flag from the downstream CLI tool, if we always want them to confirm. That's a good question for support team.

Reply:
Last, I checked support team wanted to get ack and IMO it's safe to get ack every time.